### DISCUSSION TITLE
1.20.1 Many crashes

### DISCUSSION THREAD

**Discussion by zc1108aaa** [FabricMC/fabric-api#4067] [FAQ]
![image](https://github.com/user-attachments/assets/26ee460d-d7df-435c-a9a9-7e0f317de293)
 crashes
![image](https://github.com/user-attachments/assets/7b1b22ae-6297-4534-8fff-eeac176b5cc8)
conflict
![image](https://github.com/user-attachments/assets/1a3cd272-cf9d-4c37-b491-a35472fe089d)
And these
was only released last year,
Many, hasn't been updated for a year

**Comment by Poopooracoocoo** [comment]
Got a crash report and log?

**Reply by zc1108aaa** [reply]
Many have been deleted
